src/main/java/com/easypost/service/CarrierAccountService.java

```diff
@@ -125,6 +142,8 @@ public void delete(String id) throws EasyPostException {
     private static String selectCarrierAccountCreationEndpoint(final String carrierAccountType) {
         if (Constants.CarrierAccountTypes.CARRIER_TYPES_WITH_CUSTOM_WORKFLOW.contains(carrierAccountType)) {
             return "carrier_accounts/register";
+        } else if (carrierAccountType.equals("UpsAccount")) {
```

Comment:
Need to account for "UpsMailInnovationsAccount" and
"UpsSurepostAccount" as well